example.env

```diff
@@ -1,5 +1,5 @@
 # This is an example environment file for configuring dynamic values.
-E2E_BASE_URL=http://localhost:8080/openmrs
+E2E_BASE_URL=https://dev3.openmrs.org/openmrs
```

Comment:
I'd prefer if the example setup didn't point at dev3. We need to limit the usage of dev3 for automated testing.

Reply:
Yeah, I just wanted to verify that it works with dev3. The patient registration didn't pass when running locally. It fails to load the patient-chart view after creating a patient. I'm still looking into fixing that bug. 

<img width="1009" alt="image" src="https://github.com/openmrs/openmrs-esm-patient-management/assets/27498587/8c42960f-1977-4a27-893c-974d6fb27278">

I'm going to revert the line after it is resolved.